### PR TITLE
fix: privacy info manifest set also in the PM config

### PR DESCRIPTION
**Pull request by marandaneto**
## :bulb: Motivation and Context
Fixes https://github.com/PostHog/posthog-ios/issues/111

CHecked all options here https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests

## :pencil: Checklist

- [X] I reviewed the submitted code.
- [ ] I added tests to verify the changes.
- [ ] I updated the docs if needed.
- [X] No breaking change or entry added to the changelog.

### CODE REVIEW

**Comment by marandaneto** [comment]
Screenshot 2024-02-23 at 22 35 16

File: PostHog/Resources/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeProductInteraction</string>
```
Comment (marandaneto) [inline PostHog/Resources/PrivacyInfo.xcprivacy:9]: Such as app launches, taps, clicks, scrolling information, music listening data, video views, saved place in a game, video, or song, or other information about how the user interacts with the app.

File: PostHog/Resources/PrivacyInfo.xcprivacy [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeOtherUsageData</string>
```
Comment (marandaneto) [inline PostHog/Resources/PrivacyInfo.xcprivacy:21]: > Any other data about user activity in the app.

Thinking about user screen, app opened and background, etc

File: PostHog/Resources/PrivacyInfo.xcprivacy [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+			<key>NSPrivacyCollectedDataTypeTracking</key>
+			<false/>
+			<key>NSPrivacyCollectedDataTypePurposes</key>
+			<array>
+				<string>NSPrivacyCollectedDataTypePurposeAnalytics</string>
+			</array>
+		</dict>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
+			<string>NSPrivacyCollectedDataTypeUserID</string>
```
Comment (marandaneto) [inline PostHog/Resources/PrivacyInfo.xcprivacy:33]: Technically we don't collect any id automatically, we auto generated one, but the description mention screen name which is something we do.

> Such as screen name, handle, account ID, assigned user ID, customer number, or other user- or account-level ID that can be used to identify a particular user or account.

Comment (marandaneto) [reply PostHog/Resources/PrivacyInfo.xcprivacy]: nvm, not needed

**Review by daibhin — APPROVED** [review]

**Pull request merged**